partition/roles/sonic/tasks/generate_config_db.yaml

```
@@ -0,0 +1,45 @@
+---
+- name: Create temporary directory
+  file:
+    path: /tmp/sonic
+    state: directory
+
+- name: Render input values template
+  template:
+    src: sonic-config.yaml.j2
+    dest: /tmp/sonic/sonic-config.yaml
+
+- name: Copy current config_db.json into temporary directory
+  copy:
+    src: /etc/sonic/config_db.json
+    dest: /tmp/sonic/config_db.json
+    mode: 0644
+    remote_src: true
+
+- name: Run sonic-configdb-utils
+  community.docker.docker_container:
+    name: sonic-configdb-utils
+    image: "{{ sonic_configdb_utils_image_name }}:{{ sonic_configdb_utils_image_tag }}"
+    pull: true
+    detach: false
+    cleanup: true
+    command: generate
+    volumes:
+      - /usr/share/sonic/device:/usr/share/sonic/device:ro
+      - /tmp/sonic:/etc/sonic:rw
+      - /tmp/sonic:/sonic:ro
+
+- name: Copy generated config_db.json to /etc/sonic
```

```suggestion
- name: Overwrite the existing config_db.json with the newly generated one
```
No backup of the existing one?

---

No backup. We never made backups of the existing ones.